<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftAxisWidget</class>
 <widget class="QWidget" name="ShiftAxisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shift axis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ql_referenceValue">
       <property name="text">
        <string>Reference value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="qle_referenceValue"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ql_newValue">
       <property name="text">
        <string>New value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="qle_newValue"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
